exonum/src/blockchain/mod.rs

```diff
@@ -333,26 +355,19 @@ impl BlockchainMut {
         let tx_hash = schema.block_transactions(height).object_hash();
         let error_hash = schema.call_errors(height).object_hash();
 
-        // Create block.
-        let block = Block {
+        // Add list of started services to block.
```

Comment:
Nit: Comment doesn't really fits with the following line. If the implementation of `Dispatcher::get_block_header_entries` will change, this comment will become incorrect.